src/modules/Auth/components/success/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import withTheme from 'components/shared/withTheme';
+import { PrimaryButton } from 'components/shared/toolBox/button';
+import getStyles from './styles';
+
+const SuccessScreen = ({
+  Illustration, title, description, styles, onContinue, buttonText, children
+}) => <View style={[styles.wrapper, styles.theme.wrapper]} >
+    <View style={[styles.container]} >
+      <View style={styles.illustration} >
+        <Illustration />
```

Comment:
if above is done then here can be 
```suggestion
       {illustration}
```